### PR TITLE
Completed Spacex project

### PR DESCRIPTION
**Pull request by Ntare22**
Completed Spacex project to be merged in main. Features include;
* Reserve a rocket
* Book a mission
* View missions and rockets booked in profile

### CODE REVIEW

**Review by DanSam5K — CHANGES_REQUESTED** [review]
## Hi @Ntare22  and @eapenzacharias,  ♻️ Required Changes ♻️
### Highlights 💥 💥 💥 
- [x] All linter checks are passing ✔️ 
- [x] I really want to commend your effort in trying to implement the additional feature on the profile page 👏   ✔️ 
- [x] Your project UI looks amazing ❤️  ✔️   

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It would really be nice to try adding unit tests with React Testing Library and Jest (for unit tests). What do you think? 🤔 
- [ ] I would advise that you both try to demonstrate strong collaborative culture since is a group project, by making sure that the number of your contributions through commits messages are even, what do you think?

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,87 @@
...
+## Acknowledgments
+.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
[the remark this answers is not in the document]
Comment (eapenzacharias) [reply README.md:87]: An MIT license is present at that link
https://github.com/Ntare22/spacex/blob/dev/LICENSE

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { Helmet, HelmetProvider } from 'react-helmet-async';
+import App from './App';
+import './styles/app.scss';
+import store from './redux/configureStore';
+
+const helmetContext = {};
+
```
[the remark this answers is not in the document]
Comment (eapenzacharias) [reply src/index.js:10]: Hello @DanSam5K,
Currently, the data is stored in the redux store and the reservation data stays until you refresh the page. As far as I understand we don't have to implement localStorage or session storage to save the reservation data. The store holds the data even if we change the view or use navigation.

[the remark this answers is not in the document]
Comment (DanSam5K) [reply src/index.js:10]: Hi @eapenzacharias,
I think you are right is not a must to update the localStorage or sessionStorage save the reservation or join missions. I should have made it [Optional]🤝 .
So thank you for drawing my attention to that, kindly effect the other changes and request and another re-review
Excellent job guys, you have really done a great job on your project 👍

[the remark this answers is not in the document]
Comment (eapenzacharias) [reply src/index.js:10]: @DanSam5K 
Thank you for the clarification

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
## Please go ahead and merge the branch!
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**